src/pks/flow/constitutive_relations/wrm/rel_perm_frzBC_evaluator.hh

```diff
@@ -0,0 +1,119 @@
+/*
+  Copyright 2010-202x held jointly by participating institutions.
+  ATS is released under the three-clause BSD License.
+  The terms of use and "as is" disclaimer for this license are
+  provided in the top-level COPYRIGHT file.
+
+  Authors: Ethan Coon (ecoon@lanl.gov), Bo Gao (gaob@ornl.gov)
+*/
+
+//! Evaluates relative permeability using water retention models.
```

Comment:
please update the documentation of this.  This documentation is copy-pasted from rel_perm_evaluator, which is a generic evaluator that uses WRMs to compute krel -- the WRM is the thing that evaluates Krel.  Here you evaluate Krel internally (using the omega model).  Please add:

1. a description (e.g empirical model based on ... that works in freezing conditions...)
2. a citation
3. make sure any parameters (omega, etc) are listed here as a part of the model spec.